packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -487,15 +491,23 @@ export class SDKClient {
         }
     };
 
-    private captureMetrics = (mode, type, status, cost, caller, interactingEntity) => {
+    private captureMetrics = (mode, type, status, cost, gas, caller, interactingEntity) => {
         const resolvedCost = cost ? cost : 0;
+        const resolvedGas = typeof gas === 'object' ? gas.toInt() : 0;
         this.consensusNodeClientHistorgram.labels(
             mode,
             type,
             status,
             caller,
             interactingEntity)
             .observe(resolvedCost);
+        this.consensusNodeClientHistorgram.labels(
```

Comment:
I'm unclear how we'll differentiate between the hbar cost and then the gas cost being observed.
Using the same labels will mix up items no? Did you mean to add a new metric or label?

Reply:
I agree that we should separate concerns for Spent HBar by Operator and Gas Costs incurred by user in separate metrics.